recipes/album/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "album" %}
+{% set version = "0.9.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 83117018838599ad49d98d023baa43adf4169d3e1d7a85dd4164fa23ac5e8195
+
+build:
+  number: 0
+  skip: true  # [py<37]
```

Comment:
Make that recipe `noarch: python` instead and set a minimum version on Python in host and run.